helm/templates/api/api-configmap.yaml

```diff
@@ -503,6 +503,15 @@ data:
     api: {{- toYaml .Values.api.api | nindent 6 -}}
     {{- end}}
     {{- end }}
+    # External Authentication
```

Comment:
We can use a toYaml, something like this:
{{- if .Values.api.auth }}
{{- toYaml .Values.api.auth | nindent ? -}}

Reply:
But we need to add this block only if it is enabled. Values.api.auth.external.enabled = true

Reply:
I updated the code